content/blog/2025/12/2025-12-05-twim.md

```diff
@@ -0,0 +1,293 @@
+> - Jemalloc has been repackaged with platform-specific optimizations enhancing the build. The upgrade to the dev branch of libjemalloc itself was considered as too much variability for the same release, it is planned for an upcoming release.
+> - Thanks to element-hq/synapse#18970 by dasha-uwu, we have very slightly turned down the amount of randomness when selecting join-servers, More retries also occur within a single request if necessary. Thanks to gogo199432 and lifeofguenter for reporting problems in (#128) and (#205) respectively. More opportunities are still available to make large room joins robust.
+> 
+> #### 🐞 Bug Fixes
+> 
+> - Special thanks to yefimg for fixing LDAP logout in (#231) from a report kindly made by orhtej2 in (#97); thank you for your patience waiting for domain expertise to assist here.
+> - Thanks to Radiant-Xyz the example configurations have been updated to remove allow\_check\_for\_updates. This fixes any warnings for the item no longer existing. (#221)
+> - Thanks again to Radiant-Xyz reporting in (#219) the /whoami endpoint now returns spec-compliant errors for Mautrix bridges (fe12daead9). Thanks also to @bobobo1618 for confirming the fix is working.
+> - Relations responses were sometimes incorrect in the forwards direction. This was fixed by (5147b541) bringing those responses into full compliance. Note the prior release had also fixed compliance issues but in the backwards direction.
+> - Server selection for backfill struggled sometimes for version 12 rooms. These rooms might fail to load history after join. Additional servers are now found using creators and additional\_creators instead.
+> - Room leave compliance has been fixed for an edge-case where a room becomes empty except for a locally invited user which does not have its leave event sent down /sync.
+> - Thanks to grinapo for a report which lead to the discovery of events acquired over backfill not being checked for whether they already exist.
+> 
+> #### ⏩ Upcoming
+> 
+> Several planned items could not be cut into this release: SSO/OIDC support (#7), Element Call setup assistance and documentation (#217)(#215), User-level Admin Room and Media deletion (#192), and any other assigned issue. These items are on the short-list for the next cycle and mean a lot to us. To all participants: your issues are not being ignored and we hear you.
+> 
+> #### 💜 Tuwunel Needs Your Support 💜
+> 
+> We are looking for additional sponsorship so June can work on Tuwunel again full-time. If you are a company looking to accelerate development with features you need please email june@girlboss.ceo right away.
+> 
+> ---
+> 
+> ** 💕 [GitHub](https://github.com/matrix-construct/tuwunel/) ✦ [Releases](https://github.com/matrix-construct/tuwunel/releases) ✦ [Containers](https://github.com/matrix-construct/tuwunel/pkgs/container/tuwunel) ✦ [DockerHub](https://hub.docker.com/r/jevolk/tuwunel) ✦ Tuwunel  💕**
```

Comment:
```suggestion
> **💕 [GitHub](https://github.com/matrix-construct/tuwunel/) ✦ [Releases](https://github.com/matrix-construct/tuwunel/releases) ✦ [Containers](https://github.com/matrix-construct/tuwunel/pkgs/container/tuwunel) ✦ [DockerHub](https://hub.docker.com/r/jevolk/tuwunel) ✦ Tuwunel  💕**
```

I think that fixes this issue:

<img width="1156" height="154" alt="Image" src="https://github.com/user-attachments/assets/cd47900a-1378-4907-b7da-71166f0043c2" />

Reply:
thanks, confirmed locally and pushed as [14e51c7](https://github.com/matrix-org/matrix.org/pull/3038/commits/14e51c72ee047012ce3c56a2bfa0a4bd0c8f02cf)